module/User/src/Permissions/Assertion/IsAfterMembershipEnded.php

```diff
@@ -7,8 +7,7 @@
 use Laminas\Permissions\Acl\Assertion\AssertionInterface;
 use Laminas\Permissions\Acl\Resource\ResourceInterface;
 use Laminas\Permissions\Acl\Role\RoleInterface;
-use Photo\Model\Album;
-use Photo\Model\Photo;
+use Photo\Model\{Album, Photo};
```

Comment:
Can you put these on two separate lines?

```php
use Photo\Model\{
    Album,
    Photo,
};
```

Also applies to `BaseControllerTest.php`

Reply:
Actually I undid this change at all to conflicting changes with pull #1509. The other is fixed